src/_posts/2025-01-30-new-year-new-pycon.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+author: The PyConUK team
+title: New Year, New PyCon UK!
+---
+
+{% assign links = site.data.links.md.short %}
```

Comment:
Would it make sense for us to do the assign links stuff in the general header (or at least for the blog page as a whole)? That way, we keep adding of new blog posts simpler- just copy the banner, change the title, and enter content.

Reply:
Well the `{{assign...`` in this case works because the all the links use the short form. Other blogs might want the long form, both forms or neither. 

Short answer. I am not sure, but I tend towards assuming YAGNI for now. It will be easy to make such a change if it makes sense for later posts.

Reply:
Sounds good.